lab1_1.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Бондаревой Татьяны гр. 751001</title>
+    <link rel="stylesheet" href="style.css" >
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous" >
+  </head>
+  <body>
+   <header>
+   	 <nav class="topmenu">    	
+    	<p>Navigation</p>
+    	<img src="picture/NavBlack.png" class ="nav_img_on_text" >
+    	<input type="checkbox" name="toggle" id="menu" class="toggle_menu">
+    	<label for="menu" class="toggle_menu"><i class="fas fa-bars"></i></label>    	
+    	<ul>
+    		<li><a href="">PSD WEBSITES</a></li>

```

Comment:
https://stackoverflow.com/questions/11078509/how-to-increase-the-clickable-area-of-a-a-tag-button